src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -1248,20 +1249,15 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
             else:
                 port = ports[0]
 
-        if spec.generate_cert:
+        if spec.ssl:
             san_list = spec.zonegroup_hostnames or []
-            custom_san_list = san_list + [f"*.{h}" for h in san_list] if spec.wildcard_enabled else san_list
-
-            cert, key = self.mgr.cert_mgr.generate_cert(
-                daemon_spec.host,
-                self.mgr.inventory.get_addr(daemon_spec.host),
-                custom_san_list=custom_san_list
-            )
```

Comment:
this was previously generating a certificate per RGW and setting each one up to be for the specific host it was deployed on. I admittedly can't remember why and if that was necessary (tracked it to https://github.com/ceph/ceph/commit/66e9ef16a93fc542983fb637e562278f9dc4847b but I didn't comment much on the host specific part of it). I don't think I have any recommendations or anything, but if we are changing the behavior, just something to remember if we see any auth/cert issues with RGW later.